src/Core/Application/Dependency/ManifestNormalizerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Dependency;
+
+interface ManifestNormalizerInterface
+{
+    /**
+     * @param string $type
+     * @param array<array> $configs
+     *
+     * @return array<string, array>
+     */
+    public function validateAndNormalize(string $type, array $configs): array;
```

Comment:
Breaks SRP. It shouldn't normalize anything, just validate. DTO as a part of private API is required.

Reply:
> Please, change logic according to business requirements.
Which one of the requirements was broken?
- User must have the ability to validate created task/task set/workflow by executing sdk:validate:task, sdk:validate:task-set, sdk:validate:workflow or sdk:validate:setting commands.
- Manifests validation runs automatically during sdk:init:sdk and sdk:update:sdk commands execution.
- Validation must be reusable. We should be able to run it not only from CLI.
- Tool should be able to run validation on external manifest file (eg providing route).

only sdk:validate:task, sdk:validate:task-set impossible to have 2 different validators. can be validated as a whole. Each task type can has a reference to another one.